docs/user-guides/workspace-access/remote-desktops.md

```diff
@@ -1,70 +1,26 @@
 # Remote Desktops
 
-## VNC Desktop
+## RDP
 
-The common way to use remote desktops with Coder is through VNC.
+The common way to connect to a Windows workspace is using Remote Desktop Protocol (RDP). RDP is a proprietary protocol developed by Microsoft that allows users to remotely control a computer over a network connection. It provides a graphical user interface (GUI) for remote access, enabling users to interact with the remote system as if they were sitting in front of it.
```

Comment:
This reads as patronizing to me. Remote desktop _implies_ a GUI, and given that we don't provide instructions on how to enable RDP in the workspace, I think we should assume the reader has some idea what RDP _is_.